massa-client/base_config/config.toml

```diff
@@ -9,6 +9,8 @@ private_port = 33034
 public_port = 33035
 grpc_public_port = 33037
 grpc_private_port = 33038
+# Chain id for node in sandbox mode, please update to match the target node chain id
+chain_id = 77
```

Comment:
by default we should set it to securenet for now, so that people can just download the client and use it for securenet

Reply:
Done